src/lunabot_localisation/launch/localisation.launch.py

```
@@ -1,31 +1,58 @@
+        ("rgb/camera_info", "/camera_front/camera_info"),
+    ]
+
+    return LaunchDescription(
+        [
+            # vodom
+            Node(
+                package="rtabmap_odom",
+                executable="rgbd_odometry",
+                output="screen",
+                parameters=[rtabmap_yaml, {"use_sim_time": True}],
+                remappings=camera_remappings + [("odom", "/visual_odometry")],
```

Ruff (0.14.14)

[warning] Consider `[*camera_remappings, ("odom", "/visual_odometry")]` instead of concatenation

Replace with `[*camera_remappings, ("odom", "/visual_odometry")]`

(RUF005)